docs/tutorials/basic/symbol.md

```diff
@@ -1,23 +1,73 @@
 # Symbol - Neural network graphs and auto-differentiation
 
-Besides the tensor computation interface `NDArray`, another main object in MXNet
-is the `Symbol` provided by `mxnet.symbol`, or `mxnet.sym` for short. A symbol
-represents a multi-output symbolic expression. They are composited by operators,
-such as simple matrix operations (e.g. "+"), or a neural network layer
-(e.g. convolution layer). An operator can take several input variables, produce
-more than one output variables, and have internal state variables. A variable
-can be either free, which we can bind with value later, or an output of another
-symbol.
-
-## Symbol Composition
+In a [previous tutorial](ndarray.md), we introduced `NDArray`,
```

Comment:
that link may not work on either jupyter notebook and html. need to update `mxdoc.py` to make it work